uber/config.py

```diff
@@ -572,3 +572,20 @@ def _is_intstr(s):
 # appends '../static/foo.js' to this list, that adds <script src="../static/foo.js"></script> to
 # all of the pages on the site except for preregistration pages (for performance)
 c.JAVASCRIPT_INCLUDES = []
+
+c.AWESOME_WORDS = [
```

Comment:
I'm happy to have a PROMO_CODE_STUBS variable. This variable's goal was actually to be purely a list of AWESOME_WORDS I wanted to try and compile with some strangers over time. Paul inspired me involving his BackStory™ Feel free to replace it 👍